src/webui/www/public/preferences_content.html

```diff
@@ -738,6 +738,19 @@
 }
 
 // Bittorrent tab
+updateAnonymousMode = function() {
+  if ($('anonymous_mode_checkbox').getProperty('checked')) {
+    $('upnp_checkbox').setProperty('disabled', true);
```

Comment:
this could be simpler:
```js
var isAnonMode = $('anonymous_mode_checkbox').getProperty('checked');  // should be boolean here
$('upnp_checkbox').setProperty('disabled', isAnonMode);
// and other settings
```

Reply:
I've updated the implementation for this function as well as the file's other `update` functions.